cmd/otel-allocator/main.go

```diff
@@ -93,7 +92,7 @@ func main() {
 		setupLog.Error(err, "Unable to register metrics for Prometheus service discovery")
 		os.Exit(1)
 	}
-	discoveryManager = discovery.NewManager(discoveryCtx, gokitlog.NewNopLogger(), prometheus.DefaultRegisterer, sdMetrics)
+	discoveryManager = discovery.NewManager(discoveryCtx, config.NopLogger, prometheus.DefaultRegisterer, sdMetrics)
```

Comment:
IMO we should capture these logs instead of using the Nop logger, maybe put them under debug verbosity?

Reply:
We could, but I'd do that in a separate PR. This change simply preserves current behaviour.

Reply:
sounds good to me.